BrainPortal/app/views/portal/_available_datasets_table.html.erb

```diff
@@ -41,7 +41,10 @@
     end
   %>
   <tr>
-    <td><%= overlay_description((name + "\n" + description).strip) %></td>
+    <td>
+      <%= name %>
```

Comment:
Why do you have `name` in both places?

Reply:
I kept what was in the tooltip previously.